apps/consultation-portal/screens/Subscriptions/components/SubscriptionCard/ChosenSubscriptionCard.tsx

```diff
@@ -96,7 +95,7 @@ export const SubscriptionCard = ({
           />
```

Comment:
hver er munurinn á "ChocesnSubscriptionCard" og "SubscriptionCard" ?
Sýnist þetta vera nánast sami komponentinn

Reply:
ChosenSubscriptionCard component removed since it was a duplicate of SubscriptionCard